### PR TITLE
AQI-3759

### PR DESCRIPTION
**Pull request by dedwards-AQI**
Work done to only call configs when needed and to export limits to a csv file.

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -3,15 +3,15 @@ requires = ["setuptools>=61.0"]
...
+version = "1.0.2"
 dependencies = [
     "requests",
     "protobuf==3.20.1", 
     "one_interfaces",
     "requests_cache"
     ]
 authors = [
-  { name="AquaticInformatics", email="info@aquaticinformatics.com" },
+  { name="AquaticInformatics", email="daniel.edwards@aquaticinformatics.com" },
```
Comment (KentMiller) [inline pyproject.toml:14]: Are you sure you want to use your email?

**Review by KentMiller — DISMISSED** [review]
I approve but probably wouldn't be able to point out any issues since I'm not a python developer.

**Review by joezhou-aqi — DISMISSED** [review]

File: pyproject.toml (hunk 1 above)
Comment (JonathanRobertson-AQI) [reply pyproject.toml:14]: Haha

File: pyproject.toml [hunk 2]
```diff
@@ -3,15 +3,15 @@ requires = ["setuptools>=61.0"]
 build-backend = "setuptools.build_meta"
 [project]
 name = "one_py_sdk"
-version = "1.0.1"
+version = "1.0.2"
```
Comment (JonathanRobertson-AQI) [inline pyproject.toml]: Is this a bugfix change? If it's adding functionality, I think SemVer says this should become 1.1.0.

**Pull request merged**